src/core/Song.cpp

```diff
@@ -366,7 +366,7 @@ void Song::processAutomations(const TrackList &tracklist, TimePos timeStart, fpp
 		break;
 	case PlayMode::Pattern:
 	{
-		Q_ASSERT(tracklist.size() == 1);
+		if (tracklist.size() != 1) { return; }
```

Comment:
I find this to be more descriptive.
```suggestion
		if (tracklist.empty() { return; }
```

Reply:
See, the `if (tracklist.size() != 1)` scared me because: What if we had two tracks?

Retested now, just in case, everything runs fine.